server/src/main/java/eu/fasten/server/plugins/kafka/FastenKafkaPlugin.java

```diff
@@ -258,15 +284,24 @@ public void processRecord(ConsumerRecord<String, String> record, Long consumeTim
         }
 
         // We always produce, it does not matter if local storage is enabled or not.
-        handleProducing(record.value(), consumeTimestamp);
+        handleProducing(record.value(), consumeTimestamp, priority);
     }
 
     /**
      * Writes messages to server log and stdout/stderr topics.
      *
      * @param input input message [can be null]
      */
-    public void handleProducing(String input, long consumeTimestamp) {
+    public void handleProducing(String input, long consumeTimestamp, boolean priority) {
+        String outputTopicName;
+        String errorTopicName;
+        if (priority) {
+            outputTopicName = String.format("fasten.%s.priority.out", outputTopic);
+            errorTopicName = String.format("fasten.%s.priority.err", outputTopic);
```

Comment:
Use a joint error topic for both prio and normal processing.

Reply:
Done!